tokio/Cargo.toml

```diff
@@ -41,6 +41,7 @@ full = [
   "rt-core",
   "rt-util",
   "rt-threaded",
+  "scope",
```

Comment:
For now, we probably should avoid an additional feature flag. Instead, lets bundle this up with `rt-util`.

We *will* want to initially merge this behind an "unstable" `cfg` flag.

Reply:
`rt-util` or `sync`?

Reply:
It’s a synchronization primitive, so `sync` would generally be a good choice. I mainly put it into `scope` so everything could be behind the same feature flag.

Reply:
I have to ask again what the idea is:
- Put it into module `sync ` which correlates to feature `sync`
- Put it into module `scope` and reuse feature `sync`
- Put it into module `sync` and use a new `unstable` feature

Reply:
I believe we want to:
 * Put it into module `sync` and require the `sync` future
 * _Also_ require `cfg(tokio_unstable)` (_not_ a feature flag, but a `RUSTFLAGS` cfg), as in PR #2306.

Reply:
I don't really care *where* it goes at this point. We can bikeshed it later. The key for me is:

- no new feature flags
- All work here is behind a `--cfg` flag.